### DISCUSSION TITLE
Subscriptions tab in "Your discussions" overview page

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#70022] [Discussions]
### Select Topic Area

Product Feedback

### Body

It would be cool, when I navigate to https://github.com/discussions, that there would be a tab called "Subscriptions" that would take me to something like https://github.com/notifications/subscriptions?query=is%3Adiscussion. However, subscriptions currently don't support this [search query filters](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests#about-search-terms).

**Comment by NguyenLong-02** [comment]
Same
